### DISCUSSION TITLE
OpenTTD Opt-In Automated Survey - Feedback wanted

### DISCUSSION THREAD

**Discussion by TrueBrain** [OpenTTD/OpenTTD#10632] [General]
Hi all,

For OpenTTD we are considering adding an opt-in system where you can participate in-game to automated surveys. These surveys are of course fully anonymous, and the results will be public. 

The goal of these surveys is to get more insight how our users, you, play OpenTTD.
It would also finally give us a good platform to measure popularity of NewGRFs.

I wrote a small summary of the plans here:
https://gist.github.com/TrueBrain/1a87754b5d5cc8dedceabda826acf923

I would love some input how we feel about this.

Tnx!

**Comment by PeterN** [comment]
If we can make it very obvious that it is opt-in, and also make it super easy to later opt-out (without having to dig away into settings) then it works for me.

**Comment by 2TallTyler** [comment]
For anyone following along later, the PR implementing this is #10719. 🙂
